pylearn2/scripts/tests/test_dbm.py

```diff
@@ -0,0 +1,117 @@
+"""
+Tests scripts in the DBM folder
+"""
+
+import os
+import pylearn2.scripts.dbm.show_negative_chains as negative_chains
+import pylearn2.scripts.dbm.show_reconstructions as show_reconstruct
+import pylearn2.scripts.dbm.show_samples as show_samples
+import pylearn2.scripts.dbm.top_filters as top_filters
+from pylearn2.config import yaml_parse
+from pylearn2.models.dbm.layer import BinaryVector, BinaryVectorMaxPool
+from pylearn2.datasets.mnist import MNIST
+from pylearn2.models.dbm.dbm import DBM
+from nose.tools import with_setup
+from pylearn2.datasets import control
+from pylearn2.utils import serial
+from theano import function
+from theano.compat.six.moves import cPickle
+
+
+def setup():
+    """
+    Create pickle file with a simple model.
+    tearDown is guaranteed to exexute and pop the data.
```

Comment:
It's not going to "pop the data", it's going to pop the "load_data" flag.
This also doesn't need to be in the method's docstring.

I would do:

   """
   Create pickle file with a simple model.
   """
   # tearDown is guaranteed to run push_load_data
   with open....